src/uicomponents/uirgblight.cpp

```diff
@@ -60,7 +60,7 @@ UIRGBLight::UIRGBLight(std::shared_ptr<HAEntity> _entity, lv_obj_t* _parent) :
       showColorWheel = true;
       showBrightness = true;
     }
-    if (mode == "rgb" || mode == "rgbww") {
+    if (mode == "rgb" || mode == "rgbww" || mode == "xy") {
```

Comment:
I can't test this but HA docs on https://developers.home-assistant.io/docs/core/entity/light/ say:
`The light can be dimmed and its color can be adjusted. The light's brightness can be set using the brightness parameter and read through the brightness property. The light's color can be set using the xy_color parameter and read through the xy_color property. xy_color is an (x, y) tuple.`

which would imply that we would need some changes on the way we set it?
Then again - i'm assuming you tested it and it worked. 

Those docs also state that they do some internal translations, so maybe it just works.

Reply:
indeed, from the same page

> `rgb_color`: Will be removed from the service call if not supported and translated to rgbw_color, rgbww_color, hs_color or xy_color if supported by the light.